Src/Public/Reports/vSphere/vSphere.ps1

```diff
@@ -1905,7 +1905,9 @@ foreach ($VIServer in $Target) {
                                 @{L = 'Used Space'; E = {"$([math]::Round(($_.UsedSpaceGB), 2)) GB"}}, 
                                 @{L = 'Provisioned Space'; E = {"$([math]::Round(($_.ProvisionedSpaceGB), 2)) GB"}}, 
                                 @{L = 'Changed Block Tracking Enabled'; E = {$_.ExtensionData.Config.ChangeTrackingEnabled}},
-                                @{L = 'vNICs'; E = {($_.ExtensionData.Config.Hardware.Device | Where-Object {$_ -is [VMware.Vim.VirtualEthernetCard]}).Count}}, Notes
```

Comment:
Please remove this commit as I am working on incorporating some of your suggestions, as well as others. in an upcoming update of the vSphere report.